### DISCUSSION TITLE
Streamer Mode

### DISCUSSION THREAD

**Discussion by educatalan02** [SmartlyDressedGames/Unturned-3.x-Community#3654] [Ideas]
It would be nice to add a way to check if a player is using Streamer mode for plugins.
So we can hide options/data to the client when he's streaming.

For example I got a phone plugin that adds calls. And the streamer can't open the phone because people, sees his phone number. (So with this we can prevent it)

**Comment by GazziFX** [comment]
You can add `/streamer`

**Reply by Jdance-Media** [reply]
While that is a way of doing it, it isn't very good for the streamer. Streamers tend to switch between servers frequently and wouldn't know about the command. It is so much easier for them to join the server without having to do anything.

**Reply by GazziFX** [reply]
Does any one really cares and use streamer mode?
